### ISSUE TITLE
Cannot install node-sass on node 10

### ISSUE THREAD

**Issue by Maistho** [ionic-team/ionic-app-scripts#1440]
#### Short description of the problem:
node-sass is pinned to 4.7, which is not compatible with node v10

#### What behavior are you expecting?

**Steps to reproduce:**
1. `npm i @ionic/app-scripts`
2. It crashes

**Which @ionic/app-scripts version are you using?**
`3.1.0`

**Other information:** 

Should be fixable by upgrading node-sass to v4.9.0
https://github.com/sass/node-sass/issues/2345

Some stack traces so people can google this easier:

```
 node-sass@4.7.2 postinstall /home/maistho/prog/ifiske/node_modules/node-sass
> node scripts/build.js

Building: /usr/bin/node /home/maistho/prog/ifiske/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=

...

In file included from ../../nan/nan.h:51,
                 from ../src/binding.cpp:1:
/home/maistho/.node-gyp/10.2.0/include/node/node.h:53:10: fatal error: core.h: No such file or directory
 #include "core.h"  // NOLINT(build/include_order)
          ^~~~~~~~
compilation terminated.
make: *** [binding.target.mk:115: Release/obj.target/binding/src/binding.o] Error 1
make: Leaving directory '/home/maistho/prog/ifiske/node_modules/node-sass/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/maistho/prog/ifiske/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Linux 4.16.8-1-ARCH
gyp ERR! command "/usr/bin/node" "/home/maistho/prog/ifiske/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/maistho/prog/ifiske/node_modules/node-sass
```

**Comment by Maistho** [comment]
This issue is fixed by merging PR #1435

**Comment by Asone** [comment]
PR #1435 is still awaiting merge approval. ping @dwieeb ?

**Comment by dwieeb** [comment]
Fixed by https://github.com/ionic-team/ionic-app-scripts/pull/1435

thanks for the ping @Asone

**Comment by dwieeb** [comment]
fixed in 3.1.10, which was just released
